tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -118,6 +128,21 @@ def test_repr(self):
 
 class LiveOSSourceTasksTestCase(unittest.TestCase):
     """Test the tasks of the Live OS source."""
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization.os.statvfs")
+    def test_live_os_image_size(self, statvfs):
+        """Test Live OS image size calculation."""
+        statvfs_result = Mock()
+        statvfs_result.f_frsize = 512
+        statvfs_result.f_blocks = 100
+        statvfs_result.f_bfree = 42
+        statvfs.return_value = statvfs_result
```

Comment:
Not sure if you like that better, but you can make this into one line:

`statvfs.return_value = Mock(f_frsize=512, f_blocks=100, f_bfree=42)`

Reply:
Yes, this is much better, thanks.